libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -282,7 +282,13 @@ bool AC_Fence::pre_arm_check(const char* &fail_msg) const
 {
     fail_msg = nullptr;
 
-    // if not enabled or not fence set-up always return true
+    // if fences are enabled but none selected fail pre-arm check
+    if ( enabled() && !present() ) {
+        fail_msg = "Fences enabled, but none selected";
+        return false;
+    }
+
+   // if not enabled or not fence set-up always return true
```

Comment:
Nitpick: not indented far enough.
```suggestion
    // if not enabled or not fence set-up always return true
```

Reply:
done

Reply:
🙂